tests/mocks/components.py

```diff
@@ -525,6 +525,7 @@ async def home(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optiona
         self.homed = True
         self.extra = extra
         self.timeout = timeout
+        print("called home")
```

Comment:
Was this meant to be left in?

Reply:
Yeah we should probably get rid of this, good catch @purplenicole730 !